WordPress/Classes/ViewRelated/Notifications/Style/WPStyleGuide+Notifications.swift

```diff
@@ -229,9 +226,6 @@ extension WPStyleGuide {
         fileprivate static let snippetParagraph         = NSMutableParagraphStyle(
             minLineHeight: snippetLineSize, maxLineHeight: snippetLineSize, lineBreakMode: .byWordWrapping, alignment: .natural
```

Comment:
Not really!! FWIW i've enabled Autosizing Cells, recently, and had to fall back to calculated cell heights, due to multiple UI bugs that were showing up, out of nowhere.

I still wonder why, since the SDK is flawless, as ever!